### PR TITLE
App notifications

### PR DESCRIPTION
**Pull request by lessly-zuniga**
# Description

- Notification panel with breakdown and detail is added
- Notification settings view added

Platform
- [x] iOS
- [ ] Android

Device
- [ ] Real device
- [x] Emulator

# Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] Any dependent changes have been merged and published in downstream modules

Simulator Screen Recording - iPhone 15 Pro Max - 2023-10-12 at 11 43 09

### CODE REVIEW

**Review by EdgardoV — APPROVED** [review]
There's a lot minimal fixes, Many of those I had already told you about in previous PR's.

There is some reason, why you continue to use CustomText?, for example.

File: src/screens/app/notifications/NotificationSettings/NotificationSettingsScreen.tsx [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+  const [SMSNotifications, setSMSNotifications] = useState<boolean>(true);
+  const [emailNotifications, setEmailNotifications] = useState<boolean>(true);
+  const [transactionsNotifications, setTransactionsNotifications] = useState<boolean>(true);
+  const [promotionsNotifications, setPromotionsNotifications] = useState<boolean>(true);
+  const [travelAndExpensesNotifications, setTravelAndExpensesNotifications] = useState<boolean>(true);
+
+  return (
+    <Container style={{ marginHorizontal: Theme.Sizes.Padding }}>
+      <Header title={t('notifications:titlePage')} />
+      <CustomText
```
Comment (EmmanuelSalazarID) [inline src/screens/app/notifications/NotificationSettings/NotificationSettingsScreen.tsx:19]: Use Text instead of CustomText

**Comment by EmmanuelSalazarID** [comment]
The comment above applies to several files, resolve them all

**Comment by EmmanuelSalazarID** [comment]
Simulator Screenshot - iPhone 15 Pro Max - 2023-10-12 at 12 58 23


Navigation error when tapping the notifications icon

**Pull request merged**